optaplanner-core/src/main/java/org/optaplanner/core/impl/score/stream/drools/uni/DroolsUniCondition.java

```diff
@@ -50,21 +54,22 @@
 import static org.drools.model.DSL.accFunction;
 import static org.drools.model.DSL.declarationOf;
 import static org.drools.model.DSL.on;
+import static org.drools.model.PatternDSL.from;
+import static org.drools.model.PatternDSL.pattern;
 
 public final class DroolsUniCondition<A> {
 
     private final DroolsMetadata<Object, A> aMetadata;
 
     public DroolsUniCondition(Class<A> aVariableType) {
         Declaration<A> aVariableDeclaration = declarationOf(aVariableType);
-        this.aMetadata = (DroolsGenuineMetadata) DroolsMetadata.ofGenuine(aVariableDeclaration,
-                PatternDSL.pattern(aVariableDeclaration));
+        this.aMetadata = (DroolsGenuineMetadata) DroolsMetadata.ofGenuine(aVariableDeclaration);
     }
 
     public DroolsUniCondition(Declaration<DroolsLogicalTuple> aVariableDeclaration,
             Function<Declaration<DroolsLogicalTuple>, PatternDSL.PatternDef<DroolsLogicalTuple>> patternProvider) {
         this.aMetadata = (DroolsInferredMetadata) DroolsMetadata.ofInferred(aVariableDeclaration,
-                patternProvider.apply(aVariableDeclaration));
+                () -> patternProvider.apply(aVariableDeclaration));
```

Comment:
In this case, it is not possible. (I usually do that automatically, since IDE usually points it out.)

Reply:
Strange though, because EntityDescriptor.isInitialized(Object entity) has a parameter Object too, sure there must be a way to reference it.
But oth, I would expect IntelliJ to point it out indeed. I 'll merge and see if I can change it.

Reply:
You're right.